service/telemetry.go

```diff
@@ -40,7 +40,8 @@ var (
 )
 
 type appTelemetry struct {
-	views []*view.View
+	views  []*view.View
+	server *http.Server
```

Comment:
Q. Why did we change this file that's unrelated to the rest of the PR?

A. Because this PR introduces a second test that calls `app.Start()`. Without this change, the app fails to start up on the second attempt because we never stopped listening to the `metricsAddr` port.